File: Code/Quickstarts/vb/UIProcessQuickstarts_AdvancedHostDemo/Client/PersonChangedEventArgs.vb

Imports System

Namespace Client
   
   Public Class PersonChangedEventArgs
      Inherits EventArgs
		Private objPerson As Client.EmployeeData.PersonRow

		Public Sub New(ByVal objPerson As Client.EmployeeData.PersonRow)
			Me.objPerson = objPerson
		End Sub

		Public ReadOnly Property Person() As Client.EmployeeData.PersonRow
			Get
				Return objPerson
			End Get
		End Property
	End Class

End Namespace

File: Code/Quickstarts/cs/UIProcessQuickstarts_AdvancedHostDemo/Client/PersonChangedEventArgs.cs
using System;

namespace Client
{
	public class PersonChangedEventArgs:EventArgs
	{
		Client.EmployeeData.PersonRow person;

		public PersonChangedEventArgs(Client.EmployeeData.PersonRow person)
		{
			this.person = person;
		}

		public Client.EmployeeData.PersonRow Person
		{
			get
			{
				return person;
			}
		}
	}
}
